packages/common/src/store/pages/chat/slice.ts

```diff
@@ -158,8 +160,11 @@ const slice = createSlice({
       state,
       action: PayloadAction<TypedCommsResponse<UserChat[]>>
     ) => {
+      const { summary } = action.payload
       state.chats.status = Status.SUCCESS
-      state.chats.summary = action.payload.summary
+      state.chats.summary = summary
+      state.chats.hasMore =
+        summary?.prev_count !== undefined && summary.prev_count > 0
```

Comment:
nit: `summary?.prev_count > 0` is functionally equivalent i think due to `undefined` comparisons always coming out to false, but I'm down to be explicit.

Reply:
I tried it an TS won't let me compare a potentially undefined value against 0 :shrug:

Reply:
nit: since we're checking for `prev_count > 0` we can safely do the shorthand "falsey" check `summary?.prev_count && summary.prev_count > 0`